static/bidder-info/adsinteractive.yaml

```diff
@@ -11,6 +12,6 @@ capabilities:
       - banner
 userSync:
   redirect:
-    url: "http://pb.adsinteractive.com/getuid?{{.RedirectURL}}"
+    url: "http://bid.adsinteractive.com/getuid?{{.RedirectURL}}"
```

Comment:
The redirect URL works fine for me and setting the AUID macro.
<img width="1517" alt="Screenshot 2023-05-31 at 5 34 27 PM" src="https://github.com/prebid/prebid-server/assets/8147531/4d0f3d04-fd41-46b3-8949-f2780747bd51">